### ISSUE TITLE
Ring doorbell notification not received in HomeKit

### ISSUE THREAD

**Issue by csupi003** [koush/scrypted#828]
Hello, 

A few days or weeks ago my Ring doorbell HomeKit notifications stopped working. In the Ring app I still receive it.

I added my camera to two different scrypted instances and two separate Home. None of them are receiving the notifications. 

I’m running the latest version of Scrypted and plug-in.

**Comment by blackdust217** [comment]
Seems like an issue on the Ring-side; perhaps the dev can use info from here:
https://github.com/dgreif/ring/issues/1218

**Comment by mikesalz** [comment]
For what it's worth, I am experiencing this as well.

**Comment by galoiring** [comment]
same here, after updating scrypted it started and when someone ringing I only see it in the ring app and not in the home app

**Comment by galoiring** [comment]
As far as I see, looks like it fixed over here : https://github.com/dgreif/ring/issues/1218

**Comment by koush** [comment]
Yeah this looks similar to how I use push receiver in the cloud plugin. I always suspected that this was not the right way to do push registration, but didn't bother fixing it because it was working.

**Comment by galoiring** [comment]
> Yeah this looks similar to how I use push receiver in the cloud plugin. I always suspected that this was not the right way to do push registration, but didn't bother fixing it because it was working.

Are we currently using this plugin ? (so the beta ver affected on us too?) 
Because it might be the problem to this notification that not receiving in HomeKit. 
will be glade to help if possible..

**Comment by DerLoco** [comment]
Is there an update already? Facing same issue. Motion detection is working perfectly tho.

**Comment by notlimahrelyt** [comment]
I too am having this issue.

**Comment by koush** [comment]
create a second dedicated account and share your ring to that. do not use that account with homebridge or any other projects.

**Comment by ceeeekay** [comment]
So I've had this issue since the beginning but I've been sitting tight waiting for a fix. 

I just updated the container to 0.39.0, created a new Ring account to share my camera to (because that's a great idea - thanks koush), and they all show up in the Scrypted interface under the new account, but still no motion notifications.

What am I doing wrong? Are we still waiting on an updated Ring plugin?

**Comment by mikesalz** [comment]
Not sure why this issue is closed.  I upgraded to Version 0.39.0, and also created a dedicated Ring account as suggested.  Unfortunately, doorbell ring notifications are still not being pushed to HomeKit.  Thoughts?

**Comment by csupi003** [comment]
> Not sure why this issue is closed. I upgraded to Version 0.39.0, and also created a dedicated Ring account as suggested. Unfortunately, doorbell ring notifications are still not being pushed to HomeKit. Thoughts?

Same here

**Comment by notlimahrelyt** [comment]
If this helps, I did create the dedicated account and it still wasn't working. I ended up uninstalling/reinstalling the ring plugin, setting it back up with the dedicated account again, and set "Snapshots from Prebuffer" was set to disabled (not sure if that mattered), but mine is working now.

**Comment by mikesalz** [comment]
@notlimahrelyt Thanks for the suggestion! I did the same, and it does seem to be working now.  I'll keep an eye on it to see if it continues to work...

**Comment by Almansio** [comment]
I only have one ring product, the doorbell. I removed all authorizations in my ring account, reinstalled the ring,  the ring plugin and the HomeKit plugin. No other projects is using the the ring account. Worked for a bit but i'm again not receiving any notifications anymore. Any suggestions?

**Comment by mikesalz** [comment]
@notlimahrelyt Is yours still working?  Mine stopped after 2 days.  It looks like a new issue has been opened since this one was closed: [877](https://github.com/koush/scrypted/discussions/877)

**Comment by notlimahrelyt** [comment]
Mine also stopped working today. I commented in #877

**Comment by mikesalz** [comment]
@notlimahrelyt I'm sure I am driving the devs nuts, and that is not my intention.  But there are 4 separate related issues that have all been closed.  I consolidated them all into a new one - #878 - for visibility.  I noticed that #877 was a Discussion rather than an Issue for some reason.

**Comment by GBrawl** [comment]
For me it's been working for a couple of days now. I created the dedicated account, shared my ring with it and then used the new credentials in the plugin. After that I reloaded the plugin and it's been running fine for like 3 days now.

**Comment by raiubreaksthings** [comment]
> create a second dedicated account and share your ring to that. do not use that account with homebridge or any other projects.

Can you clarify this? Create an additional ring account, share the devices to that new account but do not log into that account at all?
